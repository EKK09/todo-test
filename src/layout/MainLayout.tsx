import React from 'react';
import styled from 'styled-components';

const Title = styled.h1`
  font-size: 1.5em;
  text-align: center;
  color: palevioletred;
`;

const Wrapper = styled.div`
  max-width: 780px;
  margin: 0 auto;
`;

const MainLayout = (): React.ReactElement => (
  <Wrapper>
    <Title>TodoList</Title>
  </Wrapper>
);

export default MainLayout;
