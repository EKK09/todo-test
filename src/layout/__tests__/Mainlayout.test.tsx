import React from 'react';
import { render } from '@testing-library/react';
import MainLayout from '../MainLayout';

describe('Mainlayout component', () => {
  it('Render Mainlayout', () => {
    const wrapper = render(<MainLayout />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
