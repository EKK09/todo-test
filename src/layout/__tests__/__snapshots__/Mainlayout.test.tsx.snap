// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mainlayout component Render Mainlayout 1`] = `
<div>
  <div
    class="sc-gsTEea jpggzP"
  >
    <h1
      class="sc-bdfBQB fJsost"
    >
      TodoList
    </h1>
  </div>
</div>
`;
